nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -206,27 +238,69 @@ public void add_nullable_pass_param_simple() {
                 "-XepOpt:NullAway:SerializeFixMetadata=true",
                 "-XepOpt:NullAway:FixSerializationConfigPath=" + configPath))
         .addSourceLines(
-            "com/uber/android/Super.java",
+            "com/uber/test/Test.java",
             "package com.uber;",
             "import javax.annotation.Nullable;",
             "import javax.annotation.Nonnull;",
-            "public class Super {",
-            "   Object test(int i, Object h) {",
-            "     return h;",
+            "public class Test {",
+            "   Test () {",
+            "       // BUG: Diagnostic contains: passing @Nullable parameter 'null'",
+            "       this(null);",
             "   }",
-            "   Object test_param(@Nullable String o) {",
+            "   Test (Object o) {",
+            "      System.out.println(o.toString());",
+            "   }",
+            "",
+            "}")
+        .setExpectedFixes(
+            new FixDisplay(
+                "javax.annotation.Nullable",
+                "Test(java.lang.Object)",
+                "o",
+                "PARAMETER",
+                "com.uber.Test",
+                "com/uber/test/Test.java"))
+        .doTest();
+  }
+
+  @Test
+  public void add_nullable_param_generics() {
+    serializationTestHelper
+        .setArgs(
+            Arrays.asList(
+                "-d",
+                temporaryFolder.getRoot().getAbsolutePath(),
+                "-XepOpt:NullAway:AnnotatedPackages=com.uber",
+                "-XepOpt:NullAway:SerializeFixMetadata=true",
+                "-XepOpt:NullAway:FixSerializationConfigPath=" + configPath))
+        .addSourceLines(
+            "com/uber/Super.java",
+            "package com.uber;",
+            "import java.util.ArrayList;",
+            "class Super<T extends Object> {",
+            "   public boolean newStatement(",
+            "     T lhs, ArrayList<T> operator, boolean toWorkList, boolean eager) {",
+            "       return false;",
+            "   }",
+            "}")
+        .addSourceLines(
+            "com/uber/Child.java",
+            "package com.uber;",
+            "import java.util.ArrayList;",
+            "public class Child extends Super<String>{",
+            "   public void newSideEffect(ArrayList<String> op) {",
             "   // BUG: Diagnostic contains: passing @Nullable",
```

Comment:
Match indentation of the following line.

Reply:
b144824daae8571825ecbf1f1e72b4b81db23dca